### ISSUE TITLE
Chart pie

### ISSUE THREAD

**Issue by SwankyTigerYY** [PHPOffice/PhpSpreadsheet#1881]
This is:

```
- [ ] a feature request
```

### What is the expected behavior?

![image](https://user-images.githubusercontent.com/24326903/108931284-5b07dc80-7682-11eb-8fb3-1e9164215711.png)

### What is the current behavior?

![image](https://user-images.githubusercontent.com/24326903/108931553-c81b7200-7682-11eb-90f2-a5732b2e42a3.png)

### What are the steps to reproduce?

```
//How to use the third parameter,What does it affect.
$dataSeriesValues = [
            new DataSeriesValues(DataSeriesValues::DATASERIES_TYPE_NUMBER, 'Worksheet!$C$' . $row_start . ':$C$' . $row, null, count($pie_data)),
        ];
```

### Which versions of PhpSpreadsheet and PHP are affected?

"phpoffice/phpspreadsheet": "^1.13"，
“php”: "7.2.10"

**Comment by SwankyTigerYY** [comment]
Why is there no answer

**Comment by oleibman** [comment]
Closing. Duplicate of #2077, fixed by #2930.
